discord/guild.py

```diff
@@ -29,39 +30,9 @@ class GuildLimit(NamedTuple):
 
 
 class Guild(DiscordObject):
-    __slots__ = (
```

Comment:
why is almost the entirety of guild.py removed?

Reply:
Since the `__init__` of `Guild` was overridden pydantic was giving errors